apache/trafficcontrol
# Remove commented out PATCH /jobs code, api_capability, and docs

## What does this PR (Pull Request) do?


The PATCH method was not included and should not be referenced by
documentation or exist as an api_capability. In addition, commented-out
code in the codebase is unnecessary cruft that makes the codebase harder
to search and understand. If a PATCH method needs to be supported in the
future, the commented-out code can be retrieved from the git history
instead.

- [x] This PR fixes is not related to any Issue 


## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops

## What is the best way to verify this PR?

1. Verify traffic_ops_golang still builds and the unit tests pass
2. Make sure the documentation still builds
3. Run `db/admin seed` to re-run the `seeds.sql` file on TODB, make sure it succeeds

## The following criteria are ALL met by this PR
